Resources/Prototypes/Catalog/Fills/Boxes/security.yml

```
@@ -133,5 +133,26 @@
       amount: 9
   - type: Sprite
     layers:
-    - state: box_security
-    - state: forensic
+      - state: box_security
+      - state: forensic
```

```suggestion
    - state: box_security
    - state: forensic
```
yaml convention
same below